projects/fsastorefrontlib/src/cms-components/my-account/quote/quote-comparison/quote-comparison.component.html

```diff
@@ -0,0 +1,262 @@
+<ng-container *ngIf="quotesLoaded$ | async; else loading">
+  <!-- <ng-container *ngIf="categoryConfig; else noQuotes"> -->
+  <div class="position-relative text-center">
+    <a
+      class="h4 vertical-center-left"
+      routerLink="/my-account/my-financial-applications/"
+      routerLinkActive="active"
+    >
+      <span class="fas fa-arrow-left"></span
+    ></a>
+    <h2 class="heading-headline p-3 mt-3 mb-5 text-center">
+      <span>
+        {{
+          'quote.quotesComparison'
+            | cxTranslate
+              : {
+                  category:
+                    'fscommon.category.' + categoryConfig?.categoryCode
+                    | cxTranslate
+                }
+        }}
+      </span>
+      <br />
+      <span>
+        {{ quoteCodes.join(' / ') }}
```

Comment:
We should think of other way to display quotes here. We should avoid array methods in the template.